cms/djangoapps/contentstore/tasks.py

```diff
@@ -94,7 +132,7 @@ def rerun_course(source_course_key_string, destination_course_key_string, user_i
     # catch all exceptions so we can update the state and properly cleanup the course.
     except Exception as exc:  # pylint: disable=broad-except
         # update state: Failed
-        CourseRerunState.objects.failed(course_key=destination_course_key)
+        # CourseRerunState.objects.failed(course_key=destination_course_key)
```

Comment:
What was your intention with removing this? There are some tests failing expecting this to be recorded.